<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScenarioManagementDialog</class>
 <widget class="QWidget" name="ScenarioManagementDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QWidget" name="wgAvailable" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="lbAvailable">
            <property name="text">
             <string>Available:</string>
            </property>
            <property name="buddy">
             <cstring>twAvailable</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTreeWidget" name="twAvailable">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>1</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="pbAddToSelected">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="pbRemoveFromSelected">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lbSelected">
         <property name="text">
          <string>Selected:</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring>twSelected</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbSelectedDetailed">
         <property name="text">
          <string>(drag-and-drop scenarios onto each other in order to create a hierarchy)</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="twSelected">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::MoveAction</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectItems</enum>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="pbMoveUp">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="pbMoveDown">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbCreateScenario">
       <property name="text">
        <string comment="Create a new scenario">Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbImportScenario">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbExportScenario">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbEditScenario">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbDeleteScenario">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets/>
 <tabstops>
  <tabstop>twAvailable</tabstop>
  <tabstop>twSelected</tabstop>
  <tabstop>pbAddToSelected</tabstop>
  <tabstop>pbRemoveFromSelected</tabstop>
  <tabstop>pbMoveUp</tabstop>
  <tabstop>pbMoveDown</tabstop>
  <tabstop>pbCreateScenario</tabstop>
  <tabstop>pbImportScenario</tabstop>
  <tabstop>pbExportScenario</tabstop>
  <tabstop>pbEditScenario</tabstop>
  <tabstop>pbDeleteScenario</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
